### PR TITLE
Create ar.json

### PR DESCRIPTION
**Pull request by abugandar**
add Arabic lang to the list

### CODE REVIEW

**Comment by t3chguy** [comment]
I'm not sure but it might be better to submit translations through Weblate, so that it doesn't end up in a confused state

**Comment by MTRNord** [comment]
Also the json is broken in near all possible ways

File: src/i18n/strings/ar.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "لا يمكن تحديث إعدادات إعلام المستخدم": "إمبوسيبل دي ميتر à جور ليس نوتيفيكاتيونس وتيليساتور"،
+    "كلوز": "فيرمر"،
+    "إنشاء غرفة جديدة": "كرير أون نوفياو صالون"،
+    "لا يمكن العثور على مطابقة ماتريكس غرفة": "مستحيل دي المتاعب أون صالون ماتريكس"، "
+    "خيارات الخادم المخصصة": "خيارات دي سيرفيور بيرسوناليسيس"،
+    "حذف الاسم المستعار.": "سوبريمر l'إلياس."،
+    "هل تريد حذف الاسم المستعار للغرفة٪ (الاسم المستعار) وإزالة٪ (الاسم) s من الدليل؟": "سوبريمير l'إلياس٪ (إلياس) s دو سالون إت سوبريمر٪ (نيم) s دو ريبيرتوار؟"،
+    "الدردشة المباشرة": "المحادثة المباشرة"،
+"ديركتوري": "ريبرتوار"}
+//Add Arabic Lang
```
Comment (t3chguy) [inline src/i18n/strings/ar.json]: can't use comments

File: src/i18n/strings/ar.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{"أضف عنوان بريد إلكتروني أعلاه لتهيئة إشعارات البريد الإلكتروني": "أجوتر ون أدريس إمايل بور لا كونفيغوراتيون ديس نوتيفيكاتيونس بار إمايل"،
+    "آل مساجيس": "توس ليس مساجيس"
+    "جميع الرسائل (بصوت عال)": "رسائل ليس ليه (حصن)"،
+    "جميع الإخطارات معطلة حاليا لجميع الأهداف.": "توتيس ليس نوتيفيكاتيونس سونت ديساكتيفيس بور توس ليس أباريلز."،
+    "حدث خطأ أثناء حفظ تفضيلات إعلام البريد الإلكتروني الخاص بك.": "ون إريور إست سورفيورس دي لا سوفيغارد دي فوس بريفيرنسس دي نوتيفيكاتيونس بار e-ميل."،
+    "إلغاء إرسال": "أنولر l'إنفوا"،
+    "لا يمكن تحديث إعدادات إعلام المستخدم": "إمبوسيبل دي ميتر à جور ليس نوتيفيكاتيونس وتيليساتور"،
+    "كلوز": "فيرمر"،
+    "إنشاء غرفة جديدة": "كرير أون نوفياو صالون"،
+    "لا يمكن العثور على مطابقة ماتريكس غرفة": "مستحيل دي المتاعب أون صالون ماتريكس"، "
```
Comment (t3chguy) [inline src/i18n/strings/ar.json]: trailing open quote

File: src/i18n/strings/ar.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{"أضف عنوان بريد إلكتروني أعلاه لتهيئة إشعارات البريد الإلكتروني": "أجوتر ون أدريس إمايل بور لا كونفيغوراتيون ديس نوتيفيكاتيونس بار إمايل"،
```
Comment (t3chguy) [inline src/i18n/strings/ar.json]: You are using chars other than `,` as the entry seperator, this will not work :/

**Comment by ara4n** [comment]
@abugandar as the i18n folks have said, i really recommend following the instructions at https://github.com/vector-im/riot-web/blob/develop/docs/translating.md and using https://translate.riot.im.  Manually writing out the JSON is going to be really painful and likely to suffer from accidental errors.  Thanks for working on the translation though! :)

**Comment by abugandar** [comment]
@ara4n I start working on Translation
but add Arabic lang to main branch

**Comment by dbkr** [comment]
Closing this since it now conflicts as well, since we now have an ar.json on master.

**Pull request closed without merging**